src/sdk/model/o365/o365-restore-session.ts

```diff
@@ -877,4 +884,104 @@ export class O365RestoreSession extends Entity {
     });
   }
 
+  /**
+   * Get O365 Team Channels
+   * Proxying directly to vbo: page and pageSize are used as offset and limit respectively
+   * @param teamId
+   * @param displayName
+   * @param page
+   * @param pageSize
+   * @returns {Promise<Array<O365TeamChannel>>} the list of O365 Team Channels
+   */
+  /* istanbul ignore next: autogenerated */
+  async getTeamChannels(teamId: string, displayName: string | null = null, page?: number,
+                        pageSize?: number): Promise<Array<O365TeamChannel>> {
+    return Iland.getHttp().get(`/o365-restore-sessions/${this.uuid}/teams/${teamId}/channels`, {
+      params: {
+        displayName: displayName,
+        page: page || 0,
+        pageSize: pageSize || 50
+      }
+    }).then((response) => {
+      const json = response.data.data as Array<O365TeamChannelJson>;
+      return json.map((it) => new O365TeamChannel(it));
+    });
+  }
+
+  /**
+   * Get O365 Team Files
+   * Proxying directly to vbo: page and pageSize are used as offset and limit respectively
+   * @param teamId
+   * @param channelId
+   * @param parentId
+   * @param page
+   * @param pageSize
+   * @returns {Promise<Array<O365TeamFile>>} the list of O365 Team files
+   */
+  /* istanbul ignore next: autogenerated */
+  async getTeamFiles(teamId: string, channelId: string | null = null, parentId: string | null = null, page?: number,
+                     pageSize?: number): Promise<Array<O365TeamFile>> {
+    return Iland.getHttp().get(`/o365-restore-sessions/${this.uuid}/teams/${teamId}/files`, {
+      params: {
+        channelId: channelId,
+        parentId: parentId,
+        page: page || 0,
+        pageSize: pageSize || 50
+      }
+    }).then((response) => {
+      const json = response.data.data as Array<O365TeamFileJson>;
+      return json.map((it) => new O365TeamFile(it));
+    });
+  }
+
+  /**
+   * Get O365 Team Posts
+   * Proxying directly to vbo: page and pageSize are used as offset and limit respectively
+   * @param teamId
+   * @param channelId
+   * @param parentId
```

Comment:
@Juchazari lets add the types to all these params in your new methods please, this is what's bringing the coverage down. Also for these methods it'd be good to add a description to this parentId param for devs to know what exactly it could be, a folder, tab parent, etc...

Reply:
Updated!